src/common/filters/FilterChips.css

```diff
@@ -0,0 +1,9 @@
+.filter-chips {
+  margin-bottom: 24px;
+}
+
+.filter-chip.MuiChip-root {
```

Comment:
Better not to depend on MUI class names.

Reply:
fixed.